data/items.ts

```diff
@@ -4518,8 +4518,9 @@ export const Items: {[itemid: string]: ItemData} = {
 	},
 	quickclaw: {
 		onFractionalPriorityPriority: -2,
-		onFractionalPriority(priority, pokemon) {
+		onFractionalPriority(priority, pokemon, target, move) {
 			if (priority <= 0 && this.randomChance(1, 5)) {
+				if (move.status && pokemon.hasAbility("myceliummight")) return;
```

Comment:
```suggestion
			if (move.status && pokemon.hasAbility('Mycelium Might')) return;
			if (priority <= 0 && this.randomChance(1, 5)) {
```
just so that `randomChance` doesn't run which affects rng

Reply:
Implemented. 
Unrelated question, do you guys prefer if I rebase changes to keep one commit, or do you do that on merge?

Reply:
PRs are always squashed, no need force push everything into one commit.